### PR TITLE
Install hugegraph from github instead of download link

### PR DESCRIPTION
**Pull request by Linary**
Change-Id: I07e78cae40d7ab90640a1b7ca3bf57197c885cf1

### CODE REVIEW

File: assembly/travis/install-hugegraph.sh [hunk 1]
```diff
@@ -3,53 +3,46 @@
 set -ev
 
 if [[ $# -ne 1 ]]; then
-    echo "Must pass server version of hugegraph"
+    echo "Must pass base branch name of pull request"
```
Comment (javeme) [inline assembly/travis/install-hugegraph.sh:6]: add install-hugegraph-from-source.sh and keep install-hugegraph-from-tar.sh

**Review by javeme — DISMISSED** [review]

**Review by javeme — DISMISSED** [review]

**Comment by javeme** [comment]
ci error
image

File: assembly/travis/set-env.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+set -ev
+
+#sudo tzselect << XXG
+#5
+#9
+#1
+#1
+#XXG
+#sudo cp /usr/share/zoneinfo/Asia/Shanghai /etc/localtime
```
Comment (zhoney) [inline assembly/travis/set-env.sh:11]: delete if no use from line 5 to line 11

File: src/test/java/com/baidu/hugegraph/loader/test/functional/LoadTest.java [hunk 3]
```diff
@@ -100,4 +103,43 @@ protected static void assertContains(List<Edge> edges, String label,
...
+        long expectTimeStamp = expectDF.parse(expectDate).getTime();
+
+        DateFormat actualDF = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS");
+        actualDF.setTimeZone(actualZone);
+        long actualTimeStamp = actualDF.parse(actualDate).getTime();
+
+        Assert.assertEquals(expectTimeStamp, actualTimeStamp);
+    }
+
+
```
Comment (zhoney) [inline src/test/java/com/baidu/hugegraph/loader/test/functional/LoadTest.java:144]: delete empty line

**Review by javeme — APPROVED** [review]

File: src/test/java/com/baidu/hugegraph/loader/test/functional/LoadTest.java [hunk 4]
```diff
@@ -43,9 +43,10 @@
                                                       SERVER, PORT);
     protected static final String HTTPS_URL = String.format("https://%s:%s",
                                                             SERVER, HTTPS_PORT);
-    protected static final String PROTOCOL = "https";
-    protected static final String TRUST_STORE_FILE = "assembly/travis/conf/cacerts.jks";
-    protected static final String TRUST_STORE_PASSWORD = "changeit";
+    protected static final String HTTPS_PROTOCOL = "https";
+    protected static final String TRUST_STORE_FILE =
+                                  "assembly/travis/conf/hugegraph.truststore";
```
Comment (javeme) [inline src/test/java/com/baidu/hugegraph/loader/test/functional/LoadTest.java:48]: seems TRUST_STORE_FILE is unused

**Pull request merged**